lighthouse-core/gather/gather-runner.js

```diff
@@ -480,12 +480,18 @@ class GatherRunner {
         }
         await GatherRunner.beforePass(passContext, gathererResults);
         await GatherRunner.pass(passContext, gathererResults);
+
         if (isFirstPass) {
+          // Fetch the manifest, if it exists. Currently must be fetched before gatherers' `afterPass`.
           baseArtifacts.WebAppManifest = await GatherRunner.getWebAppManifest(passContext);
-          baseArtifacts.Stacks = await stacksGatherer(passContext);
         }
+
         const passData = await GatherRunner.afterPass(passContext, gathererResults);
 
+        if (isFirstPass) {
+          baseArtifacts.Stacks = await stacksGatherer(passContext);
```

Comment:
any reason for moving this after after pass?

Reply:
> any reason for moving this after after pass?

so we aren't tracing while it runs. The web manifest one can move out of during the trace too, but it has to be run before gatherer's `afterPass`, so we'd need a refactor on `GatherRunner.afterPass`

Reply:
maybe add a comment to this effect? it's hard to remember all the steps that happen inside those when glancing at this section :)